pkgs/development/interpreters/perl/default.nix

```diff
@@ -41,7 +41,11 @@ let
       ]
       ++ optional stdenv.isSunOS ./ld-shared.patch
       ++ optionals stdenv.isDarwin [ ./cpp-precomp.patch ./sw_vers.patch ]
-      ++ optional crossCompiling ./MakeMaker-cross.patch;
+      ++ optional crossCompiling ./MakeMaker-cross.patch
+      # https://github.com/Perl/perl5/pull/17946/
+      ++ optionals (stdenv.isDarwin && lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "11") [
```

Comment:
This patch is no longer necessary in 5.32.1 (#110654), if you rebase on the latest staging.

You'll also get LLVM 11.1 from #110631, which fixes the lldb issue.

Soon (hopefully very) #113230 will also fix Go compilation.